### PR TITLE
Add increment and decrement operators 

### PR DESCRIPTION
**Pull request by cjoshea9**
Add ++ and -- across all four supported languages and added for loop functionality in python to handle this new case

### CODE REVIEW

File: java/code_to_gast/java_router.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ def node_to_gast(node):
             return {"type": "bool", "value": 0}
         else:
             return "Unsupported prim"
-    
+    elif type(node) == str:
```
Comment (g-harel) [inline java/code_to_gast/java_router.py:22]: What is this doing?

Comment (cjoshea9) [reply java/code_to_gast/java_router.py:22]: Java doesn't have it's own type for variable names, so if we pass in just a string then we are going to assume its a variable name

Comment (g-harel) [reply java/code_to_gast/java_router.py:22]: Interesting, so what was happening before this change?

File: python/code_to_gast/py_assign.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def augop_to_str(op):
         ast.Add: "+=", 
         ast.Mult: "*=", 
         ast.Div: "/=", 
-        ast.Sub: "-=",
```
Comment (augerm) [inline python/code_to_gast/py_assign.py:42]: I think the google style guide will add this trailing comma back. At least that's what it does in JS :)

File: python/code_to_gast/py_helpers.py [hunk 3]
```diff
@@ -86,8 +86,9 @@ def pyop_to_str(op):
         ast.GtE: ">=",
         ast.Lt: "<",
         ast.LtE: "<=",
-        ast.Eq: "=="
-
+        ast.Eq: "==",
+        ast.UAdd: "++",
```
Comment (augerm) [inline python/code_to_gast/py_helpers.py:90]: I thought `++` and `--` aren't supported in python?

From my brief and very likely mistaken googling it looks like ast.UAdd might correspond with `+`. :)
https://www.programcreek.com/python/example/61006/ast.UAdd

File: python/gast_to_code/py_gast_to_code_converter.py [hunk 4]
```diff
@@ -50,14 +50,28 @@ def handle_for_range(gast, lvl=0):
...
+                incrementor = "-" + str(incrementor_value)
+            elif incrementor_op == "+=":
+                incrementor = str(incrementor_value)
+            else:
+                incrementor = "unsupported update expression"
+        
+        # ++ or -- expression
         else:
-            incrementor = "unsupported update operation"
+            if incrementor_op == "++":
```
Comment (augerm) [inline python/gast_to_code/py_gast_to_code_converter.py:67]: Will this convert `a++` --> `a += 1`?

File: test/test_loops.py [hunk 5]
```diff
@@ -43,6 +43,21 @@ def test_for_inclusiverange_increment_two(self):
         input_js_code = 'for (let i = 0; i <= 10; i += 2) {\n\t5\n}'
         expected_py_code = 'for i in range (0, 12, 2):\n\t5'
         self.assertEqual(expected_py_code, main.main(input_js_code, 'js', 'py'))
+    
+    def test_for_with_update_expression_plus(self):
```
Comment (augerm) [inline test/test_loops.py:47]: Can we add some bash and java tests?

File: java/code_to_gast/java_router.py (hunk 1 above)
Comment (cjoshea9) [reply java/code_to_gast/java_router.py:22]: In the variable assignment section it is doing the same thing just not going through the router

File: python/gast_to_code/py_gast_to_code_converter.py (hunk 4 above)
Comment (cjoshea9) [reply python/gast_to_code/py_gast_to_code_converter.py:67]: Currently for Python, we have it translating to `a++` to `++a`

File: test/test_loops.py (hunk 5 above)
Comment (cjoshea9) [reply test/test_loops.py:47]: We don't have for loops implemented in java and bash yet so can't test this use case there yet

File: python/code_to_gast/py_helpers.py (hunk 3 above)
Comment (cjoshea9) [reply python/code_to_gast/py_helpers.py:90]: After doing more research I was wrong that `++a` works in Python. It compiles on the Python ast parser but does not seem to be actually allowed in Python

**Pull request merged**